packages/extension-youtube/src/utils.ts

```diff
@@ -26,7 +26,10 @@ export interface GetEmbedUrlOptions {
   startAt?: number;
 }
 
-export const getYoutubeEmbedUrl = (nocookie?: boolean) => {
+export const getYoutubeEmbedUrl = (nocookie?: boolean, withV?:boolean) => {
```

Comment:
Not sure what `withV` means exactly - video lists? Could we find a better name for the variable

Reply:
If it's about the V in the URL, how about we describe what the V will actually do instead of just that a V was supplied?

Reply:
I change it to isPlayList，it should be more expressive.